crates/client/rpc-core/src/utils.rs

```diff
@@ -103,3 +131,184 @@ where
     let block = find_starknet_block(digest).ok()?;
     Some(block)
 }
+
+// This code was previously inside primitives/starknet/src/transaction/types.rs
```

Comment:
but this code had a feature flag `std` so it was fine no?

Reply:
Yes it should be fine because `mc-rpc-core` imports `mp-starknet` with `default-features = true`